multirowgroup.go

```diff
@@ -0,0 +1,246 @@
+package parquet
```

Comment:
To follow the naming schema of other files, this one should probably be renamed to `multi_row_group.go`.

Alternatively, the code could also be homed in `row_group.go`, up to you!